templates/cluster/openstack-standalone-cp/templates/openstackcluster.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: OpenStackCluster
+metadata:
+  name: {{ include "cluster.name" . }}
+spec:
+  {{- if .Values.apiServerLoadBalancer }}
+  apiServerLoadBalancer:
+    {{- toYaml .Values.apiServerLoadBalancer | nindent 4 }}
+  {{- end }}
+  {{- if .Values.bastion.enabled }}
+  {{- with .Values.bastion.spec }}
```

Comment:
Looks like with should be below `bastion` key and right above `spec`.

Reply:
Good catch. Updated.